dojo/db_migrations/0021_vulnerability.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 2.2.4 on 2019-10-22 21:36
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dojo', '0020_system_settings_allow_anonymous_survey_repsonse'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Vulnerability',
+            fields=[
+                ('vulnerability_id', models.CharField(help_text='Vulnerability id such as CVE-2000-0001.', max_length=200, primary_key=True, serialize=False)),
+                ('url', models.URLField(help_text='URL for more details about this vulnerability.')),
+                ('title', models.CharField(help_text='Title or summary of this vulnerability.', max_length=1000)),
+                ('description', models.TextField(blank=True, help_text='Further details of this vulnerability.')),
+                ('cwe', models.PositiveSmallIntegerField(help_text='Common weakness enumeration this vulnerability is categorized under.', null=True)),
```

Comment:
Given that some tools now give propose CWE as an array, e.g. a vulnerability could link to several CWEs, would we want to extend that directly? (probably another PR though)

Reply:
@ptrovatelli ^